tests/probes/https-outbound-probe-test.js

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+ * Copyright 2017 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+'use strict';
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+var appmetrics = (appmetrics = require('../../'));
```

Comment:
Yes, this confused me too but looking at the history there was an extra ` appmetrics =` in the original commit and I think eslint-fix or prettify must have added the brackets.